### PR TITLE
core: mermaid: Render metadata key-value pairs when drawing mermaid graph

### PR DESCRIPTION
**Pull request by nfcampos**
- if node is runnable binding with metadata attached

### CODE REVIEW

File: libs/core/langchain_core/runnables/graph.py [hunk 1]
```diff
@@ -111,7 +112,7 @@ class MermaidDrawMethod(Enum):
     API = "api"  # Uses Mermaid.INK API to render the graph
 
 
-def node_data_str(node: Node) -> str:
+def node_data_str(node: Node, *, html: bool = False) -> str:
```
Comment (vbarda) [inline libs/core/langchain_core/runnables/graph.py:115]: (nit) maybe a more descriptive name than html? `render_metadata`?

**Comment by vbarda** [comment]
could you add a test for a node w/ metadata?

File: libs/core/langchain_core/runnables/graph.py (hunk 1 above)
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/graph.py:115]: update doc-string?

What's `html`?

File: libs/core/langchain_core/runnables/graph.py [hunk 2]
```diff
@@ -111,35 +132,25 @@ class MermaidDrawMethod(Enum):
     API = "api"  # Uses Mermaid.INK API to render the graph
 
 
-def node_data_str(node: Node) -> str:
+def node_data_str(id: str, data: Union[Type[BaseModel], RunnableType]) -> str:
```
Comment (efriis) [inline libs/core/langchain_core/runnables/graph.py:135]: might be worth switching all these to be `_` prefixed to make clear they're internal

File: libs/core/langchain_core/runnables/graph.py [hunk 3]
```diff
@@ -111,35 +132,25 @@ class MermaidDrawMethod(Enum):
...
     """Convert the data of a node to a string.
 
     Args:
         node: The node to convert.
+        html: Whether to format the data as HTML rich text.
```
Comment (vbarda) [inline libs/core/langchain_core/runnables/graph.py:140]: need to update doctring here

**Pull request merged**